### PR TITLE
Updates to Storm 2.6.1

### PR DESCRIPTION
**Pull request by rzo1**
as the title says

### CODE REVIEW

File: library/storm [hunk 1]
```diff
@@ -4,6 +4,10 @@ Maintainers: The Apache Storm Project <dev@storm.apache.org> (@asfbot),
 GitRepo: https://github.com/apache/storm-docker.git
 Architectures: amd64, arm64v8
 
-Tags: 2.6.0, 2.6, latest
-GitCommit: 965de975c7af0ed35174b4276971c52965b0fa0b
+Tags: 2.6.0
+GitCommit: d1f8af26ccf800531cc773a39fac9e30efa94bca
 Directory: 2.6.0
+
```
Comment (tianon) [inline library/storm:10]: Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

```suggestion
```

Comment (rzo1) [reply library/storm:10]: TIL. Yes, I will remove it.

**Comment by rzo1** [comment]
Thanks @tianon

**Comment by tianon** [comment]
No problem, thank you!!

**Review by tianon — APPROVED** [review]

**Pull request merged**